frontends/android/BitBoxApp/app/build.gradle

```diff
@@ -1,8 +1,8 @@
 apply plugin: 'com.android.application'
 
 android {
-    buildToolsVersion "33.0.0"
-    compileSdkVersion 33
+    buildToolsVersion '34.0.0'
```

Comment:
You also should bump the toolchain in the Dockerfile like in https://github.com/digitalbitbox/bitbox-wallet-app/commit/bf04b73bc13406b4f305acc0590c040a9513559b

otherwise CI (and release) will always install it from scratch:

```
Checking the license for package Android SDK Build-Tools 34 in /opt/android-sdk/licenses                                                                                                                           
License for package Android SDK Build-Tools 34 accepted.                                                                                                                                                           
Preparing "Install Android SDK Build-Tools 34 v.34.0.0".                                                                                                                                                           
"Install Android SDK Build-Tools 34 v.34.0.0" ready.                                                                                                                                                               
Installing Android SDK Build-Tools 34 in /opt/android-sdk/build-tools/34.0.0                                                                                                                                       
"Install Android SDK Build-Tools 34 v.34.0.0" complete.
"Install Android SDK Build-Tools 34 v.34.0.0" finished.
Checking the license for package Android SDK Platform 34 in /opt/android-sdk/licenses
License for package Android SDK Platform 34 accepted.                                                     
Preparing "Install Android SDK Platform 34 (revision 3)".                                                 
"Install Android SDK Platform 34 (revision 3)" ready.
Installing Android SDK Platform 34 in /opt/android-sdk/platforms/android-34                               
"Install Android SDK Platform 34 (revision 3)" complete.
"Install Android SDK Platform 34 (revision 3)" finished.      
```

The previous time we bumped it (see linked commit above) we also bumped targetSdkVersion to the same number, but in this PR you left it at 33. Is that on purpose? Would it make sense to bump it too?

Reply:
Citing from ChatGPT:
> In summary, while the min SDK version defines the minimum Android version required to run your app, the target SDK version indicates the version against which your app's compatibility and behavior are primarily tested, and the compiled SDK version specifies the version of the Android SDK used during the compilation of your app's code.

compileSdk bump was required by some of the changes I made, iirc it was the gradle plugin that required it.
I'm not sure what `the version against which your app's compatibility and behavior are primarily tested` means in practice, but it may make sense to set it accordingly to the app usage percentages. No strong opinion, anyway.

Reply:
more info about sdks versions here: https://developer.android.com/build